src/libraries/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -259,6 +259,15 @@
     <Compile Include="Base64\Base64EncoderUnitTests.cs" />
     <Compile Include="Base64\Base64TestHelper.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Ascii\AsciiTests.Casing.cs" />
+    <Compile Include="Ascii\AsciiTests.Common.cs" />
+    <Compile Include="Ascii\AsciiTests.Equality.cs" />
+    <Compile Include="Ascii\AsciiTests.IndexOf.cs" />
+    <Compile Include="Ascii\AsciiTests.IsAscii.cs" />
+    <Compile Include="Ascii\AsciiTests.Transcode.cs" />
+    <Compile Include="Ascii\AsciiTests.Trim.cs" />
+  </ItemGroup>
```

Comment:
Shouldn't these tests be in `System.Runtime` or `System.Runtime.Extensions`?

Reply:
I guess it's correct to have them in `System.Memory`, just a bit confusing. Their APIs are part of the `System.Memory` assembly, but part of `System.Buffers.Text` namespace, but affect `Ascii` strings.